week-4/F-object-keys/exercise-part-1.js

```diff
@@ -0,0 +1,25 @@
+/*
+
+You have an object of mentors and their ages below, but the keys are inconsistent. JOSH and JAMIE are uppercase, while james and Mozafar are not. You want to return an array of mentors with the names in uppercase.
+
+You're going to have to used what we learned at the start of this lesson, and  some other JavaScript methods to accomplish this task.
+
+*/
+
+var mentorsAges = {
+  james: 29,
+  JOSH: 35,
+  JAMIE: 25,
+  Mozafar: 30
+};
+
+// ONLY EDIT BELOW THIS LINE
+
+var mentorsNames = ;
+
+var mentorsNamedCapitalised = ;
+
+// ONLY EDIT ABOVE THIS LINE
+
+console.log(mentorsNamedCapitalised);
+// Should return [ 'JAMES', 'JOSH', 'JAMIE', 'MOZAFAR' ]
```

Comment:
Again, I'd avoid saying "return". I'd go with Loic's approach of logging both expected and actual values.